### PR TITLE
fix: Add ca-certificates to docker images

### PR DESCRIPTION
**Pull request by linc01n**
Addressing #1005

### CODE REVIEW

File: distribution/docker/debian/Dockerfile [hunk 1]
```diff
@@ -1,4 +1,5 @@
 FROM debian:buster-slim
+RUN apt-get update && apt-get install -y ca-certificates
```
Comment (LucioFranco) [inline distribution/docker/debian/Dockerfile]: can we remove the apt get cache as well?

**Comment by binarylogic** [comment]
I'm curious how we could have prevented this? Is there not a way we can test this within CI? Or define a set of checks that verify Vector's environment?

**Comment by LucioFranco** [comment]
@binarylogic I'm not sure specific checks will help that much since most of these types of issues implicit things libraries need. In this case, it was missing root ca certs. I think what might be good though, in general, is to run vector with real configurations occasionally to make sure that our containers and released binaries work as intended. The way I found this bug was by actually submitting a cloud watch request to the real amazon endpoint and getting a nasty ssl error back. Maybe this is something @lukesteensen's reliability tests could find?

**Comment by binarylogic** [comment]
Thanks @linc01n !

**Pull request merged**